recipes/foursquare/init.sql

```diff
@@ -133,23 +133,24 @@ BEGIN
             SELECT 
                 date,
                 year_week, 
+                day_of_week,
                 zipcode,
                 borough, 
                 borocode,
                 category,
-                ROUND(SUM(CASE WHEN demo='Below65' AND hour='All' THEN visits END),0) AS visits_u65,
-                ROUND(SUM(CASE WHEN demo='Above65' AND hour='All' THEN visits END),0) AS visits_o65,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Morning' THEN visits END),0) AS visits_morning,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Late Morning' THEN visits END),0) AS visits_latemorning,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Early Afternoon' THEN visits END),0) AS visits_earlyafternoon,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Late Afternoon' THEN visits END),0) AS visits_lateafternoon,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Evening' THEN visits END),0) AS visits_evening,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Late Evening' THEN visits END),0) AS visits_lateevening,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Night' THEN visits END),0) AS visits_night,
-                ROUND(SUM(CASE WHEN demo='All' AND hour='Late Night' THEN visits END),0) AS visits_latenight
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='All' THEN visits END),0) AS visits_all,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Morning' THEN visits END),0) AS visits_morning,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Late Morning' THEN visits END),0) AS visits_latemorning,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Early Afternoon' THEN visits END),0) AS visits_earlyafternoon,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Late Afternoon' THEN visits END),0) AS visits_lateafternoon,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Evening' THEN visits END),0) AS visits_evening,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Late Evening' THEN visits END),0) AS visits_lateevening,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Night' THEN visits END),0) AS visits_night,
+                ROUND(SUM(CASE WHEN demo='All' AND timeofday='Late Night' THEN visits END),0) AS visits_latenight
             FROM (
                 SELECT
                     date,
+                    date_part('dow', date) as day_of_week,
```

Comment:
@AmandaDoyle  
day_of_week -> dayofweek, to be more consistent with `timeofday`?

Reply:
@mgraber @SPTKL Yes, update day_of_week to dayofweek in foursquare_daily_zipcode_timeofday and foursquare_daily_zipcode_raw